src/mono/wasm/runtime/imports.ts

```diff
@@ -10,6 +10,7 @@ import type { EmscriptenModule } from "./types/emscripten";
 
 // these are our public API (except internal)
 export let Module: EmscriptenModule & DotnetModule;
+export let anyModule: any = undefined;
```

Comment:
Can we explain what `anyModule` is/means, somewhere? I don't even see references to it in the diff

Reply:
I introduced new internal type `EmscriptenModuleInternal` instead